### DISCUSSION TITLE
Force Dark Mode Everywhere Automatically?

### DISCUSSION THREAD

**Discussion by Paft** [darkreader/darkreader#8011] [Q&A] [answered]
Am I stupid or is there not an obvious way to do this? I get tired of having to ALT+SHIFT+A on almost every site.

**Comment by alexanderby** [comment, marked as answer]
Hi @Paft! Check if you have "Not invert listed" option selected.
![image](https://user-images.githubusercontent.com/9638588/150377194-138dce6d-193d-48b9-9b65-5dc28419ac7f.png)

**Comment by Paft** [comment]
Yep, I'm stupid. Solved. Thank you.
